modules/__tests__/radium-test.js

```diff
@@ -180,4 +181,45 @@ describe('Radium blackbox tests', () => {
 
     expect(div.style.color).to.equal('green');
   });
+
+  it('applies print styles through the PrintStyle component', () => {
+    Radium(React.createClass({
+      displayName: 'TestComponent',
+
+      statics: {
+        printStyles: {
+          foo: {color: 'blue'},
+          bar: {background: 'red'}
+        }
+      },
+
+      render () {
+        return (
+          <div />
+        );
+      }
+    }));
+
+    class TestComponent2 extends Component {
+      render () {
+        return <div />;
+      }
+    }
+
+    TestComponent2.displayName = 'TestComponent2';
+    TestComponent2.printStyles = {
+      main: {color: 'black'}
+    };
+    Radium(TestComponent2);
+
+    var output = TestUtils.renderIntoDocument(<PrintStyleSheet />);
+
+    var style = React.findDOMNode(
+      TestUtils.findRenderedDOMComponentWithTag(output, 'style')
+    );
+
+    expect(style.innerText).to.equal(
+      '@media print{.Radium-TestComponent-foo{color: blue !important;}.Radium-TestComponent-bar{background: red !important;}.Radium-TestComponent2-main{color: black !important;}}' // eslint-disable-line max-len
```

Comment:
instead of disabling the lint, just break it up into multiple concatenated lines